core/src/navigation/TopNav.html

```diff
@@ -630,6 +630,11 @@
 </script>
 
 <style type="text/scss">
+  //remove default outline of all dropdowns in shellbar
+  *:focus {
```

Comment:
Is it necessary to select everything?
Will it work properly If the selector shrink to `fd-shellbar` class only?

Reply:
absolutely. Just tested and worked well. Thanks for the suggestion!